submissions/AnnaStoyano/oop-exercise/index.js

```diff
@@ -0,0 +1,112 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Mammals{
+   constructor(name,gender,friends){
+      this.species = 'mammals',
+      this.name = name,
+      this.gender = gender,
+      this.legs = 4,
+      this.friends = friends
+   }
```

Comment:
Presumably, all mammals in our world have legs. But definitely not all have 4. It's unlikely class `Mammal` can have any defaults for this.